src/NuGet.Core/NuGet.Frameworks/FrameworkNameProvider.cs

```diff
@@ -134,6 +142,11 @@ public bool TryGetShortProfile(string frameworkIdentifier, string profile, out s
             return TryConvertOrNormalize(profile, _profilesToShortName, _profileShortToLong, out profileShortName);
         }
 
+        public bool TryGetShortPlatform(string platformIdentifier, out string platformShortName)
```

Comment:
Like the other comment, I fear this API might not be a good design for future expansion